docs/api_reference/src/quantities.tex

```diff
@@ -5416,119 +5651,1485 @@
+  requires std::@\stdconcept{integral}@<typename Q::rep>
+struct uniform_int_distribution                                                         // hosted
+    : public std::uniform_int_distribution<typename Q::rep> {
+  using rep = Q::rep;
+  using base = std::uniform_int_distribution<rep>;
+
+  uniform_int_distribution();
+  uniform_int_distribution(const Q& a, const Q& b);
+
+  template<typename Generator>
+  Q operator()(Generator& g);
+
+  Q a() const;
+  Q b() const;
+
+  Q min() const;
+  Q max() const;
+};
+}
+\end{codeblock}
+
+\rSec4[qty.rand.uni.real]{Class template \tcode{uniform_real_distribution}}
+
+\indexlibraryglobal{uniform_real_distribution}
+\begin{codeblock}
+namespace mp_units {
+template<@\libconcept{Quantity}@ Q>
+  requires std::@\stdconcept{floating_point}@<typename Q::rep>
+struct uniform_real_distribution                                                        // hosted
+    : public std::uniform_real_distribution<typename Q::rep> {
+  using rep = Q::rep;
+  using base = std::uniform_real_distribution<rep>;
+
+  uniform_real_distribution();
+  uniform_real_distribution(const Q& a, const Q& b);
+
+  template<typename Generator>
+  Q operator()(Generator& g);
+
+  Q a() const;
+  Q b() const;
+
+  Q min() const;
+  Q max() const;
+};
+}
+\end{codeblock}
+
+\rSec3[qty.rand.bern]{Bernoulli distributions}
+
+\rSec4[qty.rand.bern.bin]{Class template \tcode{binomial_distribution}}
+
+\indexlibraryglobal{binomial_distribution}
+\begin{codeblock}
+namespace mp_units {
+template<@\libconcept{Quantity}@ Q>
+  requires std::@\stdconcept{integral}@<typename Q::rep>
+struct binomial_distribution                                                            // hosted
+    : public std::binomial_distribution<typename Q::rep> {
+  using rep = Q::rep;
+  using base = std::binomial_distribution<rep>;
+
+  binomial_distribution();
+  binomial_distribution(const Q& t, double p);
+
+  template<typename Generator>
+  Q operator()(Generator& g);
+
+  Q t() const;
+
+  Q min() const;
+  Q max() const;
+};
+}
+\end{codeblock}
+
+\rSec4[qty.rand.bern.negbin]{Class template \tcode{negative_binomial_distribution}}
+
+\indexlibraryglobal{negative_binomial_distribution}
+\begin{codeblock}
+namespace mp_units {
+template<@\libconcept{Quantity}@ Q>
+  requires std::@\stdconcept{integral}@<typename Q::rep>
+struct negative_binomial_distribution                                                   // hosted
+    : public std::negative_binomial_distribution<typename Q::rep> {
+  using rep = Q::rep;
+  using base = std::negative_binomial_distribution<rep>;
+
+  negative_binomial_distribution();
+  negative_binomial_distribution(const Q& k, double p);
+
+  template<typename Generator>
+  Q operator()(Generator& g);
+
+  Q k() const;
+
+  Q min() const;
+  Q max() const;
+};
+}
+\end{codeblock}
+
+% Should this come before the above, like in the C++ standard?
+\rSec4[qty.rand.bern.geo]{Class template \tcode{geometric_distribution}}
```

Comment:
See the comment.
In the C++ Standard, `geometric_distribution` comes between
`binomial_distribution` and `negative_binomial_distribution`.